layouts/shortcodes/full-changelog.html

```diff
@@ -43,21 +61,32 @@
       {{- end -}}
     </select>
   </div>
+  <div>
+    <label for="productArea">Product area</label>
+    <select class="selectorFilter" name="data-product-area" id="productArea">
+      <option value="all">Select...</option>
+      {{- range $productAreas -}}
+      {{- with . }}
+      <option value="{{ . | lower }}">{{.}}</option>
+      {{- end -}}
+      {{- end -}}
+    </select>
+  </div>
   <div class="searchInput">
     <label for="searchInput">Text search</label>
     <input type="text" id="searchInput" placeholder="Search..." />
   </div>
 </div>
 
 <div id="changelogContent">
-{{- range $changelogRendered -}}
+{{- range $index, $entry := $changelogRendered -}}
```

Comment:
```suggestion
{{- range $entry := $changelogRendered -}}
```
Nit: No need to instantiate an `$index` variable if we're not using it (it is optional).
This applies to other `range` occurrences.